torch/_dynamo/symbolic_convert.py

```diff
@@ -949,6 +949,61 @@ def COMPARE_OP(self, inst):
                     **options,
                 )
             )
+        elif (
+            isinstance(left, BaseListVariable)
+            and all(
+                isinstance(l, (ConstantVariable, DynamicShapeVariable))
+                for l in left.items
+            )
+            and isinstance(right, BaseListVariable)
```

Comment:
This is wrong, you'll report true if you compare a list and a tuple but these are never equal 

```
>>> [1] == (1,)
False
```